src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div data-qa="card" class="card">
+      <img
+        class="product-image card__product-image"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+      <h1 class="device-name card__device-name">
+        Моноблок APPLE A1419 iMac  27" Retina 5K (MNED2UA/A)
+      </h1>
+
+      <p class="product-code card__product-code">
+        Код товара: 195434
+      </p>
+
+      <div class="feedback card__feedback">
+        <div class="rating">
+          <div class="rating__point"></div>
+          <div class="rating__point"></div>
+          <div class="rating__point"></div>
+          <div class="rating__point"></div>
+          <div class="rating__point"></div>
+        </div>
+
+        <div class="comments">
+          Отзывов: 5
+        </div>
+      </div>
+      <div class="price-field card__price-field">
+        <div class="price-field__label">Цена:</div>
+        <div class="price-field__price">69 999 грн</div>
+      </div>
+
+      <form
+        data-qa="hover"
+        action="#"
+        class="buy-field"
+      >
+        <input
+          type="button"
+          class="buy-field__button"
+          value="КУПИТЬ"
+          name="buy"
+        >
+      </form>
```

Comment:
No understand, what meaning does it carry. Because the form does't submit anything. And it even hasn't method.